app/_hub/kong-inc/openid-connect/index.md

```diff
@@ -2800,32 +2800,32 @@ well.
-[Paypal](https://www.paypal.com/)                                                | [Docs](https://developer.paypal.com/docs/integration/direct/identity/log-in-with-paypal/) / [Discovery](https://www.paypal.com/.well-known/openid-configuration) / [Keys](https://api.paypal.com/v1/oauth2/certs)
-[Salesforce](https://www.salesforce.com/f)                                       | [Docs](https://developer.salesforce.com/page/Inside_OpenID_Connect_on_Force.com) / [Discovery](https://test.salesforce.com/.well-known/openid-configuration) / [Keys](https://test.salesforce.com/id/keys)
-[Yahoo!](https://www.yahoo.com/)                                                 | [Docs](https://developer.yahoo.com/oauth2/guide/openid_connect/) / [Discovery](https://api.login.yahoo.com/.well-known/openid-configuration) / [Keys](https://login.yahoo.com/openid/v1/certs)
+Provider                                                                         | Information  | Kong integration guide |
+---------------------------------------------------------------------------------|:------------:|:----------------------:|
+[Amazon AWS](https://aws.amazon.com/) [Cognito](https://aws.amazon.com/cognito/) | [Docs](https://aws.amazon.com/cognito/getting-started/) / [Discovery](https://cognito-idp.us-east-2.amazonaws.com/us-east-2_XEXO5iUSL/.well-known/openid-configuration) / [Keys](https://cognito-idp.us-east-2.amazonaws.com/us-east-2_XEXO5iUSL/.well-known/jwks.json) | [Guide](https://docs.konghq.com/enterprise/latest/plugins/oidc-cognito/) |
+[Auth0](https://auth0.com/)                                                      | [Docs](https://auth0.com/docs/protocols/oidc) / [Discovery](https://demo.auth0.com/.well-known/openid-configuration) / [Keys](https://demo.auth0.com/.well-known/jwks.json) | [Guide](https://docs.konghq.com/enterprise/latest/plugins/oidc-auth0) |
+[Google](https://www.google.com/)                                                | [Docs](https://developers.google.com/identity/protocols/OpenIDConnect) / [Discovery](https://accounts.google.com/.well-known/openid-configuration) / [Keys](https://www.googleapis.com/oauth2/v3/certs) | [Guide](https://docs.konghq.com/enterprise/latest/plugins/oidc-google/) |
+[Microsoft](https://www.microsoft.com/) Azure AD                                 | [Docs](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-v2-protocols-oidc) / [Discovery](https://login.microsoftonline.com/organizations/v2.0/.well-known/openid-configuration) / [Keys](https://login.microsoftonline.com/organizations/discovery/v2.0/keys) | [Guide](https://docs.konghq.com/enterprise/latest/plugins/oidc-azuread) |
+[Microsoft](https://www.microsoft.com/) Live Connect                             | Docs / [Discovery](https://login.live.com/.well-known/openid-configuration) / [Keys](https://nexus.passport.com/public/partner/discovery/key) ||
```

Comment:
Maybe instead of leaving these cells blank, add something like "none", "--", or even "TBD".